store/mongo/datastore_mongo.go

```diff
@@ -460,6 +460,58 @@ func (db *DataStoreMongo) UpdateDeviceGroup(ctx context.Context, devId model.Dev
 	} // to check the update count
 }
 
+func (db *DataStoreMongo) UpdateDevicesGroup(
+	ctx context.Context,
+	devIDs []model.DeviceID,
+	group model.GroupName,
+) (int64, int64, error) {
+	database := db.client.Database(mstore.DbFromContext(ctx, DbName))
+	collDevs := database.Collection(DbDevicesColl)
+
+	filter := bson.M{
+		"_id": bson.M{"$in": devIDs},
```

Comment:
we have this:
[DbDevId              = "_id"](https://github.com/mendersoftware/inventory/blob/master/store/mongo/datastore_mongo.go#L45)